src/ForwardLindbladSolver.jl

```diff
@@ -0,0 +1,73 @@
+"""
+    Exponential solver
+
+input:
+- L: the whole propagate operator
+- t_span: where the function will be evaluated,
+- rho0_vec: initial states
+- initial_type: decide we aer given an initial density matrix or a state vector
+
+output:
+- solutions at t_span
+"""
+using LinearAlgebra
```

Comment:
Dependencies should go in the GLOQ.jl file

There cannot be any space between the comment and the function header